explainaboard/tasks.py

```diff
@@ -70,6 +73,10 @@ class TaskCategory:
                  [Task(TaskType.aspect_based_sentiment_classification, True, ["F1score", "Accuracy"])]),
     TaskCategory("text-pair-classification", "predicting a class of two texts",
                  [Task(TaskType.text_pair_classification, True, ["F1score", "Accuracy"])]),
+    
+    TaskCategory("kg-link-tail-prediction", "predicting the tail entity of missing links in knowledge graphs",
+                 [Task(TaskType.kg_link_tail_prediction, True, ["Hits"])]),  
+    
 ]
 
```

Comment:
You have already made a good achievement. It would be nice if following items could be complete:
- [ ] add unittest: we usually add unittest script for several reasons: (1) check if the code could work (2) it would be easier for others to know how to play with the code [example](https://github.com/neulab/ExplainaBoard/blob/main/explainaboard/tests/test_summarization.py)
- [ ] add `docs` to introduce how to analyze this task (or dataset): here is one [example](https://github.com/neulab/ExplainaBoard/blob/main/docs/task_extractive_qa.md)

Reply:
Added basic unit tests and docs